### PR TITLE
Added downloader support

### PR DESCRIPTION
**Pull request by ravi5175**
What's New
- Added downloader support
- Added md5_hash_checksum support for model corruption

Bug Fixes
- Fixed minor bugs 

Improvements
- Increased Efficiency with new model support
- ASCII colored code support added for better visibility

### CODE REVIEW

**Review by ravi5175 — COMMENTED** [review]
Tested on Local host GPU "Nvidia GTX 1050 Ti"
Tested on Local host CPU "Intel i5 8th generation"

File: freshlybuiltimagejaano/downloader.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                        self.hash_signature_match(model_name,available_models,dir_path,status_code)
+                        return self.status_code
+                
+        else: 
+            print("no reference found for "+model_name)
+            self.status_code=1001
+            return status_code
+    
+    
+    def start_downloading(self,model_name,dir_path,status_code):
```
Comment (vishal2612200) [inline freshlybuiltimagejaano/downloader.py]: Please add a docstring to the function and class,

**Review by vishal2612200 — CHANGES_REQUESTED** [review]

**Review by ravi5175 — COMMENTED** [review]
New DocString added to downloader

File: freshlybuiltimagejaano/downloader.py (hunk 1 above)
Comment (ravi5175) [reply freshlybuiltimagejaano/downloader.py]: done

**Review by vishal2612200 — APPROVED** [review]

**Pull request merged**